stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Yeah, adding `stack.yaml` doesn't seem to belong in a PR whose topic is IRCv3 message tag support and should probably be a separate PR. :)